lua/gruvbox_crisp/palette.lua

```
@@ -0,0 +1,89 @@
+  func     = "#ff79c6",  -- pink for functions
+  func_green = "#b8bb26", -- green for LaTeX commands and entity names
+  number   = "#bd93f9",
+  type     = "#8be9fd",  -- cyan for types/classes
+  operator = "#ffb86c",
+  variable = "#f8f8f2",
+  decorator= "#9b4fa0",
+  const     = "#d3869b", -- non-numeric constants / enum members
+  preproc   = "#fe8019", -- preprocessor/meta
+  kw_ctrl   = "#fb4934", -- control keywords (if/for/return, storage)
+  property  = "#83a598", -- fields/properties/parameters
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)